packages/@webex/plugin-meetings/src/controls-options-manager/index.ts

```diff
@@ -133,6 +134,40 @@ export default class ControlsOptionsManager {
     this.setLocusUrl(options?.locusUrl);
   }
 
+  /**
+   * Set controls for this meeting.
+   *
+   * @param {Array<ControlConfig>} controls - Spread Array of ControlConfigs
+   * @returns {Promise<Array<any>>}- Promise resolving if the request was successful.
+   */
+  protected update(...controls: Array<ControlConfig>) {
```

Comment:
will ultimately replace `setControls()`, but is disconnected for now [independent workflow/stack], while still be accessible from downstream.